lib/url.c

```diff
@@ -3349,7 +3349,8 @@ static void conn_meta_freeentry(void *p)
 
 static CURLcode create_conn(struct Curl_easy *data,
                             struct connectdata **in_connect,
-                            bool *async)
+                            bool *async,
+                            bool *use_slist)
```

Comment:
I don't think you need to update this variable in the function, but you can just pass in the value:

```suggestion
                            bool use_slist)
```

Reply:
U need to update because we are using that bool to check whether or not an error happened AFTER we used the resolution 

The logic being everything after we parse slist would be related at least in part to it.

I think it needs a rename and we need another output varible? I am not sure what the best way to do it.